test/OpenFeature.Contrib.Hooks.Otel.Test/MetricsHookTest.cs

```diff
@@ -0,0 +1,163 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using OpenFeature.Model;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using Xunit;
+
+namespace OpenFeature.Contrib.Hooks.Otel.Test
+{
+    public class MetricsHookTest
+    {
+        [Fact]
+        public void After_Test()
+        {
+            // Arrange metrics collector
+            var exportedItems = new List<Metric>();
+            Sdk.CreateMeterProviderBuilder()
+                .AddMeter("*")
+                .ConfigureResource(r => r.AddService("openfeature"))
+                .AddInMemoryExporter(exportedItems, option => option.PeriodicExportingMetricReaderOptions = new PeriodicExportingMetricReaderOptions { ExportIntervalMilliseconds = 100 })
+                .Build();
+
+            // Arrange
+            const string metricName = "feature_flag.evaluation_success_total";
+            var otelHook = new MetricsHook();
+            var evaluationContext = EvaluationContext.Empty;
+            var ctx = new HookContext<string>("my-flag", "foo", Constant.FlagValueType.String, new ClientMetadata("my-client", "1.0"), new Metadata("my-provider"), evaluationContext);
+
+            // Act
+            var hookTask = otelHook.After(ctx, new FlagEvaluationDetails<string>("my-flag", "foo", Constant.ErrorType.None, "STATIC", "default"), new Dictionary<string, object>());
+            // Wait for the metrics to be exported
+            Thread.Sleep(150);
+
+            // Assert
+            Assert.True(hookTask.IsCompleted);
+
+            // Assert metrics
+            Assert.NotEmpty(exportedItems);
+
+            // check if the metric is present in the exported items
+            var metric = exportedItems.FirstOrDefault(m => m.Name == metricName);
+            Assert.NotNull(metric);
+
+            var noOtherMetric = exportedItems.All(m => m.Name == metricName);
```

Comment:
Could you enhance / add some some assertions here about the attributes [here](https://github.com/open-feature/dotnet-sdk-contrib/pull/114/files#diff-39cf92a95324d98f7eae3aac643b31749e76c89e4cd6223b6ed39c40381050daR78-R81)?